proposals/0304-structured-concurrency.md

```diff
@@ -609,17 +606,17 @@ A new, detached task can be created with the `detach` operation. The resulting t
 ```swift
 /// Create a new, detached task that produces a value of type `T`.
 @discardableResult
-static func detach<T: Sendable>(
+public func detach<T>(
   priority: Task.Priority = .unspecified,
-  operation: @escaping @concurrent () async -> T
+  operation: @Sendable @escaping () async -> T
 ) -> Task.Handle<T, Never>
 
 /// Create a new, detached task that produces a value of type `T` or throws an error.
 @discardableResult
-static func detach<T: Sendable>(
+public func detach<T, Failure>(
   priority: Task.Priority = .unspecified,
-  operation: @escaping @concurrent () async throws -> T
-) -> Task.Handle<T, Error>
```

Comment:
here the proposal is correct and the impl is in the wrong; thanks for reminding me though!